RTE Block Configuration and usage (#5562)
umbraco/UmbracoDocs: 12/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/configuration.md

![Rich Text Editor Inline mode](../../built-in-property-editors/rich-text-editor/images/inline-mode-new.png)

## Blocks

> Blocks can be added as Elements of the Rich Text Editor.
> The Fields configuring Blocks are described as part of the [Blocks in Rich Text Editor](rte-blocks.md) article.

Reviewer: ```suggestion
Blocks can be added as elements in the Rich Text Editor. Configuration and rendering of Blocks is described in the [Blocks in Rich Text Editor](rte-blocks.md) article.
```
